README.md

```diff
@@ -16,12 +16,52 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
+### API
+
+```bash
 npm install @stoplight/spectral
 ```
 
+### CLI
```

Comment:
It no longer seems like we need both API and CLI, as theyre the same other than the -g. 

This stuff is tough, maybe copy eslint https://github.com/eslint/eslint